musicalbum.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  def initialize(on_spotify, publish_date)
+    super()
```

Comment:
the same as previous comment

Reply:
on_spotify is the only variable needed according to the diagram and publish_date is only needed to calculate the `can_be_archived?` method. I do not see the reason to take up space with variables that are not going to be useful to the `MusicAlbum` class.